### PR TITLE
refactor: replace deprecated `TestBed.get`

### PR DESCRIPTION
**Pull request by LayZeeDK**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The deprecated `TestBed.get` method is used in the test suite and documentation.

## What is the new behavior?
The strongly typed `TestBed.inject` method is used in the test suite and documentation.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
Related: https://github.com/ngrx/platform/pull/2503
Cc @timdeschryver 

I believe the formatting changes are done by a precommit hook or something like that.

### CODE REVIEW

**Comment by LayZeeDK** [comment]
A few type-related compilation errors. I'll fix them soon.

**Comment by LayZeeDK** [comment]
> This replaces all the `TestBed.get` usages, right?

Yes, except that the ones specifically for schematics-generated code are in https://github.com/ngrx/platform/pull/2503

File: docs/effects/testing.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ describe('My Effects', () => {
       ],
     });
 
-    effects = TestBed.get(MyEffects);
+    effects = TestBed.inject(MyEffects);
```
[the remark this answers is not in the document]
Comment (LayZeeDK) [reply docs/effects/testing.md:44]: Oh 😅 I'll revert them.

**Comment by LayZeeDK** [comment]
Good to go, @timdeschryver

File: docs/effects/testing.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ import * as MyActions from '../actions/my-actions';
 
 describe('My Effects', () => {
   let effects: MyEffects;
-  let actions: ReplaySubject&lt;any&gt;;
+  let actions: ReplaySubject & lt;
```
[the remark this answers is not in the document]
Comment (LayZeeDK) [reply docs/effects/testing.md:72]: I know. How can we revert it?  🤔

[the remark this answers is not in the document]
Comment (LayZeeDK) [reply docs/effects/testing.md:72]: I might have to edit it directly in GitHub to circumvent Git hooks.

[the remark this answers is not in the document]
Comment (timdeschryver) [reply docs/effects/testing.md:72]: That, or use the `--no-verify` flag should also work I think.

```
git commit -m "foo" --no-verify
```

**Comment by LayZeeDK** [comment]
Merged your changes, resolved old docs comment, and rebased, @timdeschryver.

**Comment by timdeschryver** [comment]
Thanks again @LayZeeDK !

**Pull request merged**